### ISSUE TITLE
Add cpanfile into git repo

### ISSUE THREAD

**Issue by colinnewell** [cv-library/UUID4-Tiny#6]
It would be handy if the cpanfile was present in the git repo so that someone can develop tweaks to the module without dist::zilla.

Or is that necessary if everything is core?

**Comment by jjatria** [comment]
With only core deps, I guess having a `cpanfile` is unnecessary. But you're probably still going to need Dist::Zilla for _some_ hacking.

I find that an approach like this is the best of both worlds (added in `dist.ini`):

``` ini
[Git::CommitBuild]
release_branch = build
branch =
```

It makes `dzil release` commit the "built" version of the codebase into the `build` branch. The empty `branch` means that we _don't_ want to commit every time we do `dzil build`.

This means that someone could do

```
cpanm git://github.com/cv-library/UUID4-Tiny.git@build
```

and install from Github, which is nice.

Then again, if we are going with an approach like that (with multiple branches with the raw and minted code) we might want to move the built code to `master` and keep the raw source in a sort of "dev" branch instead.

**Comment by jjatria** [comment]
A much better approach was implemented in #11. How much we learn in only a couple of years... :smiling_face_with_tear:
